src/components/addnewtask.js

```diff
@@ -22,7 +22,7 @@ onAddClicked() {
             <div style={styles.addTask} class="row">
                 <div class="col-sm-3"></div>
                 <div class="col-sm-4"> <InputField /> </div>
-                <div class="col-sm-2"> <AddButton onClick = {this.onAddClicked}  /> </div>
+                <div class="col-sm-2"> <AddButton clickHandler = {this.onAddClicked}  /> </div>
```

Comment:
1) So I changed the prop to be called `clickHandler` just to make it clearer about which was a prop and which was the `onClick` event handler.

Here we pass a function called `onAddClicked` down to your `AddButton` as a `prop`.